docs/get-started/_index.md

```diff
@@ -1,36 +1,97 @@
-{{% card link="/get-started/quickstarts/drive-rover/" %}}
-{{< /cards >}}
+<div>
+{{< imgproc src="/viam/viam.png" alt="Viam overview" resize="800x" class="aligncenter" >}}
+</div>
 
-If you're ready to try it on your own machine, start by installing `viam-server`.
-If you need any help along the way, join our community:
+<div>
+{{< imgproc src="/viam/pet-feeder.png" alt="Machine components" resize="400x" class="alignright" >}}
+</div>
 
-<!-- markdownlint-disable-file MD034 -->
+At the core of Viam is the open-source `viam-server` executable which runs on a device and manages hardware and software for a machine.
+To use Viam with a machine, you create a configuration that contains components and services and you can choosen from a wide variety of available models for different resources.
+For example, if you have a pet-feeding machine you might install `viam-server` on a Raspberry Pi and create a machine configuration using a camera model and a servo model for the hardware you are using.
 
-{{< cards >}}
-{{% card link="/get-started/installation/" %}}
-{{% manualcard link="https://discord.gg/viam" webm_src="/heart.webm" mp4_src="/heart.mp4" alt="A robot drawing a heart" %}}
+The Viam App provides the user interface for configuring your machine.
+It allows you to configure builtin drivers or drivers from the Viam Registry.
+`viam-server` then manages and runs the drivers for the configured resources.
+As you configure your machine, you can test each resource to confirm it is working as expected.
+
+{{<gif webm_src="/test.webm" mp4_src="/test.mp4" alt="Test a camera stream" max-width="700px" class="aligncenter">}}
+
+Viam is built to be extensible, allowing you to create your own resources for custom hardware or software and deploy your code using modules through the Viam Registry.
+
+On top of configuration, the Viam App also provides several higher-level functionalities for your machine, such as:
+
+- **Data Management**: Any data on your machine can be synced to the cloud, from where you can query it using SQL, MQL, or with code.
+- **Machine Learning**: Train machine learning models on collected data and deploy ML models to machines to enable them to intelligently see and interpret the world around it.
+- **Motion planning**: Plan and move machine components.
+- **Simultaneous Localization And Mapping (SLAM)**: A machine can map its surroundings and find its position on a map.
+
+## Standardized APIs
+
+Each category of {{< glossary_tooltip term_id="resource" text="resource" >}} has a standardized API that you can access with an [SDK (software development kit)](/sdks/) in your preferred programming language.
+For example, you can send the same [`SetPower` command](/components/motor/#setpower) to any kind of motor, using any of the available SDKs.
+
+The standardized nature of the resource APIs, means that when you build machines, you can swap out components such as motors without changing any code.
```

Comment:
```suggestion
The standardized nature of Viam's resource APIs means that when you build machines, you can swap out components such as motors without changing any of code.
```
should we add something like "assuming the same method is implemented" bc they aren't always? idk

Reply:
I think that's a detail that is too early to point out to people given how introductory this content is
```suggestion
The standardized nature of Viam's resource APIs means that when you build machines, you can swap out components such as motors without changing any code.
```